test/oli/sections_test.exs

```diff
@@ -53,17 +53,14 @@ defmodule Oli.SectionsTest do
       Sections.enroll(user2.id, section.id, [ContextRoles.get_role(:context_learner)])
       Sections.enroll(user3.id, section.id, [ContextRoles.get_role(:context_learner)])
 
-      assert length(Sections.list_enrollments(section.slug)) == 3
+      enrollments = Sections.list_enrollments(section.slug)
 
-      [one, two, three] = Sections.list_enrollments(section.slug)
+      assert length(enrollments) == 3
```

Comment:
You could remove this since below it's technically asserting there are 3 items, looks good otherwise.

Reply:
That's right, good catch 👍